frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -72,6 +105,33 @@ export const ListPageCreateDropdown: React.FC<ListPageCreateDropdownProps> = req
   .ListPageCreateDropdown;
 export const ListPageFilter: React.FC<ListPageFilterProps> = require('@console/internal/components/factory/ListPage/ListPageFilter')
   .default;
+
+/**
+ * A hook that manages filter state for the ListPageFilter component.
+ * @param data - An array of data points
+ * @param rowFilters - (optional) An array of RowFilter elements that define the available filter options
```

Comment:
@TheRealJon do we want to use JSDoc's optional parameter syntax? https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

`@param {RowFilter[]} [rowFilters] an array of RowFilter elements that define the available filter options`

Reply:
When I address https://bugzilla.redhat.com/show_bug.cgi?id=2083154, I'll have to see if using this syntax would affect the format the generated docs. For now, I think it would be best to keep it simple.